src/plugins/data/public/query/timefilter/timefilter.ts

```diff
@@ -148,6 +150,14 @@ export class Timefilter {
     return _.clone(this._refreshInterval);
   };
 
+  public getMinRefreshInterval = () => {
+    return this._minRefreshInterval;
+  };
+
+  public setMinRefreshInterval = (t: number) => {
+    this._minRefreshInterval = Math.max(t, 0);
+  };
```

Comment:
I don't know if it would be desired or necessary for applications to change the `minRefreshInterval` value.

Reply:
Agreed, we can probably leave this out for now (`setMinRefreshInterval`)

Reply:
2be42e1